### DISCUSSION TITLE
Change font size printer

### DISCUSSION THREAD

**Discussion by jeffrobert** [TriliumNext/Trilium#4594] [General]
First off, thanks for everyone involved in developing Trilium, as well for those providing support on the forum.

I can print notes locally on my local Brothers printer, but cannot find a way to enlarge the printed characters. Modifying the *-font-size parameters does not affect printed size. The "scale" options is greyed out in the print menu, but I do not know it would make a difference anyway.

Is there something to be done about it?

**Comment by Openstreetmapler** [comment]
My suggestion: Read the documentation about [how to customize the print CSS](https://docs.triliumnotes.org/user-guide/concepts/notes/printing-and-pdf-export#customizing-the-print-css), and use the following code for your Print CSS (experiment around with the font size until it is right for you):

```css
body, body * {
    font-size: 16px;
}
```
